portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/controlpanel/apps/appmanager/AppManagement.testcase

```diff
@@ -735,6 +735,65 @@ definition {
 			value1 = "Error:Please upload a file with a valid extension (JAR, LPKG, or WAR File).");
 	}
 
+	@description = "This test check search apps in marketplace store and check sign in and sign out."
+	@priority = "3"
+	test SearchMarketplaceStore {
+		task ("Given user login marketplace store assert user login in Purchased page") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Marketplace",
+				panel = "Control Panel",
+				portlet = "Store");
+
+			AppManager.loginMarketplaceStore();
+
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Marketplace",
+				panel = "Control Panel",
+				portlet = "Purchased");
+
+			AssertElementPresent(
+				locator1 = "UserBar#USER_SIGN_OUT",
+				value1 = "Sign Out");
+		}
+
+		task ("Then go to marketplace store and search apps and purchase") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Marketplace",
+				panel = "Control Panel",
+				portlet = "Store");
+
+			WaitForPageLoad();
+
+			AppManager.searchApp(searchName = "No Robots");
+		}
+
+		task ("Then logout marketplace store") {
```

Comment:
Update the description from _Then logout marketplace store_ to _Then assert logout from marketplace store page also logout in Purchase page _

Reply:
done